roles/cluster_api/tasks/main.yml

```diff
@@ -28,6 +31,17 @@
     owner: root
     group: root
 
+- name: Copy over all provider configuration to the remote system
+  vexxhost.kubernetes.multi_copy:
+    src: "{{ role_path }}/files/controllers"
+    dest: "{{ cluster_api_controller_path }}"
+    mode: "0755"
+    owner: root
+    group: root
+  when:
+    - cluster_api_infrastructure_provider == "openstack"
+    - cluster_api_infrastructure_version is ansible.builtin.version('0.12.0', 'ge')
```

Comment:
I think we can YAML of the requested controller (I really don't think they're going to be introducing any new ones other than this one).  However, we'll have to switch this out to a `template` and ensure that we have a way to set the image.

One of the things about this collection is ability to do airgapped installs, so we need to be able to override the images here.